src/xstateTree.spec.tsx

```diff
@@ -262,4 +263,44 @@ describe("xstate-tree", () => {
       expect(view2).toBe(getMultiSlotViewForChildren(interpreter2, "ignored"));
     });
   });
+
+  describe("rendering a routing root inside of a routing root", () => {
+    it("throws an error during rendering", async () => {
+      const machine = createMachine({
+        id: "test",
+        initial: "idle",
+        states: {
+          idle: {},
+        },
+      });
+
+      const RootMachine = createXStateTreeMachine(machine, {
+        View() {
+          return <p>I am root</p>;
+        },
+      });
+      const Root = buildRootComponent(RootMachine, {
+        basePath: "/",
+        history: createMemoryHistory(),
+        routes: [],
+      });
+
+      const Root2Machine = createXStateTreeMachine(machine, {
+        View() {
+          return <Root />;
+        },
+      });
+      const Root2 = buildRootComponent(Root2Machine, {
+        basePath: "/",
+        history: createMemoryHistory(),
+        routes: [],
+      });
+
+      expect(async () => {
+        render(<Root2 />);
+
+        await delay(10);
```

Comment:
Does it need this delay?

Reply:
Yes because of https://github.com/koordinates/xstate-tree/blob/master/src/xstateTree.tsx#L507-L510
Could probably be fine with just `await delay()` but without it no error happens

Reply:
Ah what about `jest.runAllTimers();` then? It would make the tests go marginally faster